lib/components/bm-confirm-location-on-map.js

```diff
@@ -0,0 +1,148 @@
+/* @flow */
+'use strict'
+
+const geolocation = require('@blinkmobile/geolocation')
```

Comment:
I used to be super diligent about wrapping things in Angular services. I'll remind myself how to do this and consider it for next time. :)